api/src/main/java/org/apache/iceberg/Transaction.java

```diff
@@ -137,6 +137,14 @@ default AppendFiles newFastAppend() {
    */
   DeleteFiles newDelete();
 
+  /**
+   * Create a new {@link UpdateTableStatistics update table statistics API} to add or remove statistics
+   * files in this table.
+   *
+   * @return a new {@link UpdateTableStatistics}
+   */
+  UpdateTableStatistics updateTableStatistics();
```

Comment:
Usually we add new methods to Table and Transaction at the same time.

Reply:
while working on https://github.com/apache/iceberg/pull/4741 I didn't realize i should have changes in Table as well. sure, will add.